contrib/opencensus-ext-azure/opencensus/ext/azure/common/storage.py

```diff
@@ -109,7 +114,7 @@ def __exit__(self, type, value, traceback):
     def _maintenance_routine(self, silent=False):
         try:
             if not os.path.isdir(self.path):
-                os.makedirs(self.path)
+                os.makedirs(self.path, exist_ok=True)
```

Comment:
Can you put a comment explaining the reasoning for this? (raise condition when `os.makedirs` is called after the `if` check passes but the file was already created by another worker).